zerver/tests/test_subs.py

```diff
@@ -117,18 +117,20 @@ def test_make_stream_public(self):
 
         do_change_is_admin(user_profile, True)
         params = {
-            'stream_name': 'private_stream'
+            'stream_name': ujson.dumps('private_stream'),
+            'is_private': ujson.dumps(False)
         }
-        result = self.client_post("/json/make_stream_public", params)
+        result = self.client_patch("/json/streams/private_stream", params)
         self.assert_json_error(result, 'You are not invited to this stream.')
 
         self.subscribe_to_stream(email, 'private_stream')
 
         do_change_is_admin(user_profile, True)
         params = {
-            'stream_name': 'private_stream'
+            'stream_name': ujson.dumps('private_stream'),
```

Comment:
Shouldn't it make more sense if you initialized the dictionary and then used `ujson.dumps` when you are passing it to `client_patch`?